File: backend/router/es_map.go
package router

// ES mapping 路由
func (this *WebServer) runEsMap() {

	group := this.engine.Group("ES映射操作", "/api/es_map")
	{
		group.POST(false, "查询映射列表", "/ListAction", this.esMappingController.ListAction)
		group.POST(true, "修改Es映射", "/UpdateMappingAction", this.esMappingController.UpdateMappingAction)
	}
}

File: frontend/src/api/es-map.ts
import {request} from "@elasticview/plugin-sdk";

const api = '/api/es_map/'

export function ListAction(data) {
  return request({
    url: api + 'ListAction',
    method: 'post',
    data
  })
}

export function UpdateMappingAction(data) {
  return request({
    url: api + 'UpdateMappingAction',
    method: 'post',
    data
  })
}

